test/test_nn_quantized.py

```diff
@@ -34,6 +36,78 @@ def test_relu_api(self):
         self.assertEqual(qY, qY_hat)
 
 
+class DynamicModuleAPITest(QuantizationTestCase):
+    @given(
+        batch_size=st.integers(1, 5),
+        in_features=st.integers(16, 32),
+        out_features=st.integers(4, 8),
+        use_bias=st.booleans(),
+    )
+    def test_linear_api(self, batch_size, in_features, out_features, use_bias):
+        """test API functionality for nn.quantized.dynamic.Linear"""
+        W = torch.rand(out_features, in_features).float()
+        W_scale, W_zp = _calculate_dynamic_qparams(W, torch.qint8)
+        W_q = torch.quantize_linear(W, W_scale, W_zp, torch.qint8)
+        X = torch.rand(batch_size, in_features).float()
+        B = torch.rand(out_features).float() if use_bias else None
+        qlinear = nnqd.Linear(in_features, out_features)
+        qlinear.set_weight(W_q)
+        # Simple round-trip test to ensure weight()/set_weight() API
+        self.assertEqual(qlinear.weight(), W_q)
+        W_pack = qlinear._packed_weight
+        qlinear._packed_weight = W_pack
```

Comment:
What's the purpose of these two lines?

Reply:
Oops...This might be added during some rebase...Fixed in https://github.com/pytorch/pytorch/pull/24408.